mailpoet/lib/Config/AccessControl.php

```diff
@@ -13,7 +12,7 @@ class AccessControl {
   const PERMISSION_MANAGE_SUBSCRIBERS = 'mailpoet_manage_subscribers';
   const PERMISSION_MANAGE_FORMS = 'mailpoet_manage_forms';
   const PERMISSION_MANAGE_SEGMENTS = 'mailpoet_manage_segments';
-  const PERMISSION_MANAGE_AUTOMATIONS = Engine::CAPABILITY_MANAGE_AUTOMATIONS;
+  const PERMISSION_MANAGE_AUTOMATIONS = self::PERMISSION_MANAGE_EMAILS;
```

Comment:
We seem to be quite granular in the roles in general (managing subscribers, forms, segments, ...). Seeing that, isn't removing the automations role a bit of an inconsistency? Wouldn't it be better to just add the `editor` role as a default (same as for emails and for plugin admin)?

Reply:
> Wouldn't it be better to just add the editor role as a default (same as for emails and for plugin admin)?

~~Tbh, I tried that. For some reason, it did not work as expected~~

Ahhh! Capabilties are updated during plugin update and not on runtime! Yes, I agree, that would be better. Preparing...